snippets/assert-attribute-must-be-set.js

```diff
@@ -0,0 +1,48 @@
+/* Usage of this snippet:
+ *   In the case when you want to confirm an element having a specified attribute.
```

Comment:
どういうときに使うの方ではなく、このコードが何をするのかに焦点を当てた方がわかりやく感じるんですがどうでしょうか？
```suggestion
 * Detect if the specified element has the specified attribute or not.
```

Reply:
スニペットはコードが書けない人向けという側面が強いので、
「コードが何をするか」
を書いても
「それを使って何ができるか」
がわからない可能性があります。（コーディングできる人には伝わるでしょうが）

適切な目的で使ってもらわないと意味がないため
「こういうときにこのスニペットを使ってください」
という書き方にしています。

Reply:
例えば公開済みのスニペットについて、さゆちゃんやあらきさんが
「どのスニペットをどんなときに使うべきか」
をなかなか理解できておらず、一般ユーザーにおいてもこの問題は同様となります。

Reply:
エンジニアさんの視点だとこのコードが何をするのかを記載した方がわかりやすいかもしれませんが、私のような非エンジニアさんだと、どういう時に使うスニペットかがないと使い方がわからないかもです…